app/src/pages/outside/loginPage/socialSection/newsBlock/postBlock/postBlock.scss

```
@@ -36,3 +53,153 @@
+  font-family: $FONT-SEMIBOLD;
+  color: $COLOR--charcoal-grey;
+}
+
+.post-handle,
+.post-separator,
+.post-date {
+  color: $COLOR--steel-grey;
+}
+
+.post-content {
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)